idunn/blocks/description.py

```diff
@@ -0,0 +1,48 @@
+from enum import Enum
+from typing import Literal, Optional
+
+import idunn
+from idunn import settings
+from idunn.blocks.wikipedia import WikipediaBlock
+from .base import BaseBlock
+
+DESC_MAX_SIZE = int(settings["DESC_MAX_SIZE"])
+
+
+def limit_size(content: str) -> str:
+    if len(content) > DESC_MAX_SIZE:
+        return content[:DESC_MAX_SIZE] + "…"
+
+    return content
```

Comment:
This function is not used, as far as I can see.

Reply:
:man_facepalming: 

I'll just apply it to all description results, call it on all description outputs, which will be consistent with former Wikipedia data and extend the same rule to pagejaunes and OSM (even though it won't make a huge difference for these two as their description field is usually short).